unattended_scripts/install_functions/opendistro/common.sh

```diff
@@ -3,7 +3,7 @@ repobaseurl="https://packages.wazuh.com/4.x"
 
 getConfig() {
     if [ -n "${local}" ]; then
-        cp ./$config_path/$1 $2
+        cp "$(pwd)/$config_path/$1" $2
```

Comment:
Maybe it will be better to use `realpath $0` to obtain the path to the script and remove the script name with sed.

Reply:
Used a variable to store the base path of the main script, this allow to run the unattended from different paths

Done in https://github.com/wazuh/wazuh-packages/pull/1046/commits/cb80bbeb99ab7a9aee91d27be8dd4004610c0b10